pages/partials/navbarWorkspace.ejs

```diff
@@ -0,0 +1,41 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-info align-items-center" style="height:55px">
```

Comment:
Rather than making this a partial, I suggest just putting it all directly into `pages/workspace/workspace.ejs`. It's only going to be used in that single file, so having a partial doesn't really make sense?